src/components/Layout/Header/NavBar/ExpandedMenuItem.tsx

```diff
@@ -0,0 +1,85 @@
+import { ChevronRightIcon, ExternalLinkIcon } from '@chakra-ui/icons'
+import { Badge } from '@chakra-ui/layout'
+import { MenuItem } from '@chakra-ui/menu'
+import { MenuItemProps } from '@chakra-ui/menu/dist/declarations/src/menu'
+import { Link, useColorModeValue } from '@chakra-ui/react'
+import { ThemeTypings } from '@chakra-ui/styled-system'
+import { ColorProps } from '@chakra-ui/styled-system/dist/declarations/src/config/color'
+import { CSSProperties } from 'react'
+import { RawText } from 'components/Text'
+
+type ExpandedMenuItemProps = {
+  label: string | undefined
+  value?: string
+  valueDisposition?: 'positive' | 'neutral' | 'negative' | 'info'
+  badge?: string
+  badgeColor?: ThemeTypings['colorSchemes']
+  hasSubmenu?: boolean
+  isDisabled?: boolean
+  externalUrl?: string
+} & MenuItemProps
+
+export const ExpandedMenuItem = ({
+  label,
+  value,
+  badge,
+  badgeColor = 'grey',
+  hasSubmenu = false,
+  valueDisposition = 'neutral',
+  isDisabled = false,
+  externalUrl = undefined,
+  ...props
+}: ExpandedMenuItemProps) => {
+  const blackShade = useColorModeValue('blackAlpha.600', 'whiteAlpha.600')
+  const greenShade = useColorModeValue('green.600', 'green.500')
+  const redShade = useColorModeValue('red.800', 'red.500')
+  const yellowShade = useColorModeValue('yellow.600', 'yellow.200')
+
+  const valueColor: ColorProps['color'] = (() => {
+    switch (valueDisposition) {
+      case 'positive':
+        return greenShade
+      case 'negative':
+        return redShade
+      case 'info':
+        return yellowShade
+      case 'neutral':
+      default:
+        return blackShade
+    }
+  })()
+
+  const disabledStyleOverride: CSSProperties = { cursor: 'auto', opacity: 1 }
+
+  const expandedMenuItem = (
+    <MenuItem
+      display='flex'
+      alignItems='center'
+      px={3}
+      {...props}
+      closeOnSelect={!hasSubmenu}
+      isDisabled={isDisabled}
+      style={isDisabled ? disabledStyleOverride : undefined}
+    >
+      <RawText flex={1}>{label}</RawText>
```

Comment:
I'd prefer that this be a `<Text>` component and that `label` is a string pointing to a translation. Is there a need for `label` to be dynamic text and not translated?

Reply:
It was only to keep `ExpandedMenuItem` extensible, but I can also see value in forcing the caller to have defined translations.

Updated in https://github.com/shapeshift/web/pull/1341/commits/08b4791f902eaf6700816c015425629378f59bc2.